scripts/devops_tasks/build_packages.py

```diff
@@ -31,7 +31,7 @@ def build_packages(targeted_packages, distribution_directory, is_dev_build=False
         print("Generating Package Using Python {}".format(sys.version))
         run_check_call(
             [
-                "python",
+                sys.executable,
```

Comment:
Discovered when running locally that python calling python doesn't nest properly unless you're using system PATH updates like how the VSAgents do them.

Using sys.executable is the proper way to ensure that you're always in the same environment.